tools/src/main/java/org/apache/kafka/tools/ShareConsumerPerformance.java

```diff
@@ -153,6 +157,9 @@ private static void consume(List<KafkaShareConsumer<byte[], byte[]>> shareConsum
             // Preserve interrupt status
             Thread.currentThread().interrupt();
         }
+        for (Future<?> future : futures) {
+            future.get(1, TimeUnit.MILLISECONDS);
```

Comment:
Is there any point in waiting for 1 ms here? Wouldn't `future.get()` be as effective?